opba-protocols/xs2a-protocol-tests/xs2a-bdd-wiremock/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/wiremock/mocks/WiremockAccountInformationRequest.java

```diff
@@ -48,7 +48,7 @@ public SELF open_banking_redirect_from_aspsp_ok_webhook_called_for_api_test() {
 
         // no consent UI for embedded test
         this.redirectOkUri = consentInitiateRequest.getHeader(TPP_REDIRECT_URI).replace("/embedded-server/", "/");
-        ExtractableResponse<Response> response = RestAssured
+        ExtractableResponse<Response> response = withSignatureHeaders(RestAssured.given())
```

Comment:
This function is called within the user session (from UI), not Fintech - it is protected only by users' cookie and not by request signing - no signature headers here

Reply:
Removed signature header adding for non-fintech requests